bin/pynani.py

```diff
@@ -205,33 +175,35 @@ def on_modified(self, event):
     observer = Observer()
     observer.schedule(event_handler, path=PATH, recursive=True)
     observer.start()
+    
     print("Hot Reload is now Hot!")
     print("Now watching for file changes with WatchDog\n")
+    
     try:
         while True:
             time.sleep(1)
     except KeyboardInterrupt:
         observer.stop()
+    
     observer.join()
 
-def server_start(port):
-    PORT = port if port == "" else 8000
-
-    web_dir = os.path.abspath(path(os.path.dirname( __file__ ), '..', 'core', "public"))
+def server_start(port: str):
+    PORT = int(port) if port else 8000
+    web_dir = os.path.abspath(path(os.path.dirname(__file__), "..", "core", "public"))
 
     os.chdir(web_dir)
 
     handler = http.server.SimpleHTTPRequestHandler
 
     with socketserver.TCPServer(("", PORT), handler) as httpd:
         print(f"Server now started at localhost: {PORT}")
-        print("PyNani version that you're currently using is 1.0.0-alpha")
-        print("To quit the server or to turn off exit the command-line")
+        print(f"PyNani version that you're currently using is {VERSION}")
+        print("To stop the server, press CTRL+C or exit the command-line")
```

Comment:
CTRL+C to kill the server will never work. The Thread is stopping it from working. It is forever (of course if you close the file)